### DISCUSSION TITLE
Autopilot stats

### DISCUSSION THREAD

**Discussion by MarcFinns** [teslamate-org/teslamate#3870] [Ideas]
Not sure if the info is accessible via api, but it would be cool if statistics about autopilot utilization could be added. For example, which sections of a drive were manual vs auto.

**Comment by ygx0** [comment]
Sadly, Tesla doesn't share this at all via their APIs but it would be great to see. One work around might be having a CAN/OBD reader that is reading and forwarding, but that's too much work for many people.

**Reply by MarcFinns** [reply]
They do collect that information though, as they know in case of accident, whether the auto pilot was engaged or not. What a pity they don't share it via API!

**Reply by ygx0** [reply]
Correct they do, Tesla Insurance has access to it too. But that’s all, no public access.

**Reply by epijimenez** [reply]
This might have changed recently since Tessie seems to be offering that distinction on their drives data.

**Comment by JakobLichterfeld** [comment]
It is available via Tesla Fleet API, which has lots of drawbacks (see https://docs.teslamate.org/docs/guides/api#why-tesla-fleet-and-telemetry-only-have-drawbacks) and therefore is not used.

**Reply by arcastro** [reply]
@JakobLichterfeld, does it have to be one or the other? 

Is it possible to continue to use Owner API for most features and only leverage Fleet API to track autopilot/FSD?
